deployments/local-setup-environment/provisioning/install-nodejs.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e
+
+echo "Installing Node.js and NPM..."
+
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+
+echo 'export NVM_DIR="$HOME/.nvm"' >> ~/.profile
+echo '[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"' >> ~/.profile
+source ~/.profile
+
+nvm install 14.3.0
+nvm install 11.15.0
```

Comment:
Can you please remind me why do we need two different versions? Something about tests requiring node 14 rings in my head but I am not sure... I think it's worth adding a comment about it to this script.

Reply:
`npm install` for `keep-core` and `keep-ecdsa` is failing if `node > 11.15.0` is used. In the contrary, e2e tests use some features (like `json-modules`) which are available in `node > 13` hence we have this awful version switching. There are some plans for the future to repair this.

Reply:
Can you please open an issue in `keep-core` and `keep-ecdsa` about `npm` version and reference this issue from here? Just short information in a comment with a link to an issue should be enough.

Reply:
Sure thing.

Reply:
I've opened an issue in `local-setup` repo: https://github.com/keep-network/local-setup/issues/14